<launch>
  <!--
  <node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd_to_pointcloud" args="$(find pcl_tutorials)/data/table_scene_mug_stereo_textured.pcd 1" output="screen">
    <remap from="/cloud_pcd" to="/full_cloud_filtered" />
  </node>
  -->
  
  <!--
  <node pkg="point_cloud_converter" type="point_cloud_converter" name="point_cloud_converter" output="screen" respawn="true" >
    <remap from="/points_in" to="/shoulder_cloud"/>
  </node>
  -->

  <include file="$(find table_objects)/launch/get_table_objects.launch"/>
  
  <!-- Cluster the points -->
  <node pkg="nodelet" type="nodelet" name="table_object_clusters" args="load pcl/EuclideanClusterExtraction pcl_manager" output="screen"> 
    <remap from="~input" to="extract_objects_indices/output" /> 
    <rosparam> 
      max_queue_size: 10 
      cluster_tolerance: 0.03 
      spatial_locator: 0 
      cluster_min_size: 20 
      max_clusters: 10 
    </rosparam> 
  </node> 

  <node pkg="pcl_ros" type="pointcloud_to_pcd" name="pointcloud_to_pcd" output="screen" respawn="true">
    <remap from="/input" to="table_object_clusters/output" />
  </node>
  
</launch>

